src/hello.py

```diff
@@ -1,6 +1,15 @@
 def main():
     print("Hello from workspace!")  # noqa: T201
 
+    a: str = 1
```

Comment:
🚫 **[pyright]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Type "Literal[1]" is not assignable to declared type "str"
  "Literal[1]" is not assignable to "str" (reportAssignmentType)
<!-- __reviewdog__:ChA2YmU0MzExZjZhZmQxZDU1EgdweXJpZ2h0 -->